src/processTargets/getMarkStage.ts

```diff
@@ -10,7 +10,7 @@ import { MarkStage } from "./PipelineStages.types";
 export default (mark: Mark): MarkStage => {
   switch (mark.type) {
     case "cursor":
-      return new CursorStage(mark);
+      return new CursorStage();
```

Comment:
Hmm I'm hesitant to move away from the uniformity here, as we may turn this case statement into a map

Slightly mixed feelings about the empty function thing, as it can be useful. I guess forcing the comment saying it's empty on purpose is maybe not a bad thing either way?

Cc/ @AndreasArvidsson

Reply:
I agree and would prefer to keep the uniformity.